### ISSUE TITLE
Laravel form builder

### ISSUE THREAD

**Issue by RERE110** [nWidart/laravel-modules#699]
Hello,
Your package is great!
I have a small question or rather suggestion is it possible to add kristijanhusak/laravel-form-builder which I think could greatly improve the modularity of the modules :).
Cordially

**Comment by nWidart** [comment]
Hello,

Thank you, glad you enjoy it! 😄 
Regarding the package, I don't believe this should be "forced" by this package.
Not every module needs this for example. I prefer leaving this open for everyone to chose which dependencies they want.

**Comment by RERE110** [comment]
Actually I was so limited that I forgot the basics of programming ;)

**Comment by ntdorji** [comment]
Hi, I have some issue with kristijanhusak/laravel-form-builder. I have created 2 Modules: Home and Office. I started with Home, everything was working fine including the forms which i created using the form-builder. 

But when I later created the module Office, the get/post actions from my forms in Home module redirects to Office controller. I am getting the error: Action Modules\Office\Http\Controllers\HomeController@index not defined.

So I was wondering if @RERE110 ran into similar issues and how it was resolved. Or is there some configuration that I have missed??
